libkbfs/key_bundle_v2.go

```diff
@@ -266,22 +265,31 @@ func (wkg TLFWriterKeyGenerationsV2) IsWriter(user keybase1.UID, deviceKID keyba
 }
 
 // ToTLFWriterKeyBundleV3 converts a TLFWriterKeyGenerationsV2 to a TLFWriterKeyBundleV3.
-//
-// TODO: Add a unit test for this.
 func (wkg TLFWriterKeyGenerationsV2) ToTLFWriterKeyBundleV3(
-	ctx context.Context, codec kbfscodec.Codec, crypto cryptoPure,
-	keyManager KeyManager, kmd KeyMetadata) (
+	codec kbfscodec.Codec, crypto cryptoPure,
+	tlfCryptKeyGetter func() ([]kbfscrypto.TLFCryptKey, error)) (
 	TLFWriterKeyBundleV2, TLFWriterKeyBundleV3, error) {
 	keyGen := wkg.LatestKeyGeneration()
 	if keyGen < FirstValidKeyGen {
 		return TLFWriterKeyBundleV2{}, TLFWriterKeyBundleV3{},
 			errors.New("No key generations to convert")
 	}
 
+	wkbV2 := wkg[keyGen-FirstValidKeyGen]
+	for u, dkimV2 := range wkbV2.WKeys {
```

Comment:
Can you add a comment above here to make it clear the loop is just performing a check?  At some point, we might want to go through and conditionalize things like this for performance (but not right now).